shell_sort/ShellSort.go

```diff
@@ -0,0 +1,29 @@
+package main
+
```

Comment:
comment on exported function ShellSort should be of the form "ShellSort ..."

*GoLintBear, severity NORMAL, section `golint`.*